tuned/plugins/plugin_scheduler.py

```diff
@@ -642,29 +652,39 @@ def _instance_apply_static(self, instance):
 		sched_cfg = [(option, str(value).split(":", 4)) for option, value in instance._scheduler.items()]
 		buf = [(option, self._convert_sched_cfg(vals))
 				for option, vals in sched_cfg
-				if re.match(r"group\.", option)
+				if re.match(r"^group\.", option)
```

Comment:
Why this change? IMHO the re.match should already match from the beginning of the string.

Reply:
You're right. I'll remove the change.